Assets/UniRx.Async/PlayerLoopHelper.cs

```diff
@@ -62,38 +66,30 @@ public static class PlayerLoopHelper
         static ContinuationQueue[] yielders;
         static PlayerLoopRunner[] runners;
 
-        static PlayerLoopSystem[] InsertRunner(PlayerLoopSystem loopSystem, Type loopRunnerYieldType, ContinuationQueue cq, Type loopRunnerType, PlayerLoopRunner runner)
+        static PlayerLoopSystem[] InsertRunner(PlayerLoopSystem loopSystem, Type loopRunnerYieldType,
+            ContinuationQueue cq, Type loopRunnerType, PlayerLoopRunner runner)
         {
+            EditorApplication.playModeStateChanged += (state) =>
```

Comment:
This part also has to be under `#if UNITY_EDITOR`

Reply:
It's right, thank you, i've updated the pull request